python-sdk/tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb

```diff
@@ -0,0 +1,580 @@
+        "aciconfig = AciWebservice.deploy_configuration(cpu_cores = 1, \n",
+        "                                               memory_gb = 1, \n",
+        "                                               tags = {'area': \"digits\", 'type': \"automl_classification\"}, \n",
+        "                                               description = 'sample service for Automl Classification')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from azureml.core.webservice import Webservice\n",
+        "from azureml.core.model import Model\n",
+        "\n",
+        "aci_service_name = 'automl-databricks-local'\n",
+        "print(aci_service_name)\n",
+        "aci_service = Model.deploy(ws, aci_service_name, [model], inference_config, aciconfig)\n",
+        "aci_service.wait_for_deployment(True)\n",
+        "print(aci_service.state)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Test the Best Fitted Model\n",
+        "\n",
+        "#### Load Test Data - you can split the dataset beforehand & pass Train dataset to AutoML and use Test dataset to evaluate the best model."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "dataset_test = Dataset.Tabular.from_delimited_files(path='https://dprepdata.blob.core.windows.net/demo/crime0-test.csv')\n",
+        "\n",
+        "df_test = dataset_test.to_pandas_dataframe()\n",
+        "df_test = df_test[pd.notnull(df_test['Primary Type'])]\n",
+        "\n",
+        "y_test = df_test[['Primary Type']]\n",
+        "X_test = df_test.drop(['Primary Type', 'FBI Code'], axis=1)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Testing Our Best Fitted Model\n",
+        "We will try to predict digits and see how our model works. This is just an example to show you."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "fitted_model.predict(X_test)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "### Delete the service"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "aci_service.delete()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "![Impressions](https://PixelServer20190423114238.azurewebsites.net/api/impressions/MachineLearningNotebooks/how-to-use-azureml/azure-databricks/automl/automl-databricks-local-with-deployment.png)"
+      ]
+    }
+  ],
+  "metadata": {
+    "authors": [
+      {
```

Comment:
Do we still need this metadata?

Reply:
not sure. It was there in previous github location as well.

Reply:
I checked other automl notebooks and author is required.